src/OrchardCore/OrchardCore.Abstractions/BackgroundTasks/IBackgroundTaskEventHandler.cs

```diff
@@ -0,0 +1,11 @@
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace OrchardCore.BackgroundTasks
+{
+    public interface IBackgroundTaskEventHandler
+    {
+        Task ExecutingAsync(BackgroundTaskContext context, CancellationToken cancellationToken);
+        Task ExecutedAsync(BackgroundTaskContext context, CancellationToken cancellationToken);
+    }
```

Comment:
Some docs would be useful here.

Reply:
Okay, will do

Reply:
@Piedone

Yes would be useful, and i wanted to complete the doc around the new `IBackgroundTaskEventHandler`, but just saw that there is nothing at all, e.g. on how to define / register / configure an `IBackgroundTask`.

So maybe in a separate PR for a more complete doc on `ModularBackgroundService`, `IBackgroundTask`, and then on the new `IBackgroundTaskEventHandler`.

Reply:
I think at this point just a couple of XML comments here, i.e. on the interface and the two methods, would be sufficient. What this interface is about, when are the methods called. To give people a basic understanding.

Reply:
Okay cool, will do it